packages/modules/create-imodel/src/components/create-imodel/CreateIModel.tsx

```diff
@@ -78,6 +78,14 @@ export type CreateIModelProps = {
 };
 
 export function CreateIModel(props: CreateIModelProps) {
+  return (
+    <ThemeProvider>
```

Comment:
sure its needed here? base component has it already at the root. what does this do? is it for toaster?
lets remove then from base theme provider?

Reply:
This is needed because of our use of `useToaster` in the original component, since these component can be used directly, they need to be wrapped. Now I question the need of wrapping our component at all since the styles need to be imported by the application anyway, they already need to add the potential complexity of importing aliased pacakge... We could advertize ourselves as "Like iTwinUI component, our components need to be wrapped in a iTwinUI V3 ThemeProvider". I dont like delegating our problems to the user, but since this is how iTwinUI must be used, we might be adding more issues by duplicating ThemeProviders like we do for these components.

Reply:
i think i agree with the idea of removing ThemeProvider 🤔 in v2, it was needed because packages were creating "boundaries", but v3 doesn't have that problem (v2 and v3 can overlap) so it should be fine to make the application responsible for it.

this whole update should probably have gone into admin-components v2 though.

Reply:
however this may not be realistic. i don't think react context works once you use an alias, because it is a "different" package.

Reply:
> As these packages are different, the context should work as they are imported from their packages.

What I mean is that the following are imported from "different" packages, so they will likely not use the same context.

```tsx
// in admin-components
import { ThemeProvider } as "@itwin/itwinui-react"; // v3
```
```tsx
// in application
import { ThemeProvider as ThemeProviderV3 } as "itwinui-react-v3";
```

But I haven't tested, so this should be verified